CHANGELOG.md

```diff
@@ -1,6 +1,8 @@
 # v 2.6.0 (?)
 Changes in this release:
-
+- Add `dryrun_all` method to  the `project` fixture. Does a dryrun on every resource of a project. Also does some sanity checks.
+- Add `dryrun_and_deploy_all` method to the `project` fixture. Dryruns, deploys and does a final dryrun on every resource of a project. Also does some sanity checks.
```

Comment:
What is the purpose of the first dryrun?

Reply:
The purpose of the first dryrun is to check if there is the `purged` key in the results. Meaning that the resource is not currently deployed and is being created. To be honest, if `assert_create_or_delete` is `False`I don't think there is much use in the first dryrun (maybe checking if it fails before the deploy?)